content/en/opentelemetry/interoperability/environment_variable_support.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Configuring Tracing Libraries Using OpenTelemetry Environment Variables
```

Comment:
Let's rename "Tracing Libraries" to SDK's. In Stuart's OTel drop-in docs PR, his title is `Using OpenTelemetry Instrumentations with Datadog SDKs`. So perhaps the title could be:
- `Using OpenTelemetry Environment Variables with Datadog SDKs`, or
- `Configuring Datadog SDKs with OpenTelemetry Environment Variables`